charts/primary-site/values.yaml

```diff
@@ -61,6 +61,16 @@ inboxListener:
       # - name: AWS_COPY_PART_SIZE_BYTES
       #   value: 104857600
 
+    serviceAccount:
+      enabled: false
+      annotations: {}
+      ## Service accounts are not required by default. You can use them on AWS to allow
```

Comment:
Is this specific to AWS or is this also how one would do this on GCP?

Reply:
I think once we support this, it would make sense to recommend doing the same thing there